src/ResultWriter/EnergyOutput.cpp

```diff
@@ -549,6 +571,10 @@ void EnergyOutput::printEnergies() {
                       << 2.0 / 3.0 * std::log10(energiesStorage.plasticMoment()) - 6.07 << " ,"
                       << ratioPlasticMoment;
       }
+      if (totalMomentumX || totalMomentumY || totalMomentumZ) {
```

Comment:
Should be an `&&` to make sure that all of these three quantities exist.

Reply:
There might be cases where only one of them or two of them are zero and the remainging quantities are non-zero. For example, if we take a travelling wave scenario in x-direction. We get a zero in y and z directions and a non-zero momentum in x direction. I guess it still makes sense to have all the three values printed in any case.